doc/modules/databases/index.rst

```diff
@@ -10,6 +10,12 @@ identifiers.org mappings and URLs (:py:mod:`indra.databases.identifiers`)
 .. automodule:: indra.databases.identifiers
     :members:
 
+Bioregistry mappings and URLs (:py:mod:`indra.databases.bioregistry`)
```

Comment:
Maybe rename to `bioregistry_client` to reduce confusion when reading code, since if you see `bioregistry.some_function()` maybe it looks like the actual package